modulemd/v2/tests/test-modulemd-merger.c

```diff
@@ -100,6 +104,47 @@ merger_test_deduplicate (CommonMmdTestFixture *fixture,
   g_assert_cmpstr (baseline, ==, deduplicated);
 }
 
+static void
+merger_test_merger_with_real_world_data (CommonMmdTestFixture *fixture,
+                         gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleIndex) fedora_index = NULL;
+  g_autoptr (ModulemdModuleIndex) updates_index = NULL;
+  g_autoptr (ModulemdModuleIndex) merged_index = NULL;
+  g_autoptr (ModulemdModuleIndexMerger) merger = NULL;
+  g_autofree gchar *yaml_path = NULL;
+  g_autoptr (GPtrArray) failures = NULL;
+  g_autoptr (GError) error = NULL;
+  
+  yaml_path =
+    g_strdup_printf ("%s/modulemd/v2/tests/test_data/long-valid.yaml",
+                     g_getenv ("MESON_SOURCE_ROOT"));
+  
+  
+  fedora_index = modulemd_module_index_new ();
+  
+  yaml_path =
+    g_strdup_printf ("%s/modulemd/v2/tests/test_data/long-valid.yaml",
```

Comment:
This is the same data, making this test effectively identical to the deduplication test. It would be better to use the `f29.yaml` and `f29-updates.yaml` data files instead of `long-valid.yaml`.